src/services/WalletService.ts

```diff
@@ -201,4 +221,32 @@ export default abstract class WalletService {
         return addressListResponse;
       });
   }
+
+  private async getAccountsXpub(accountIdx: number): Promise<void> {
```

Comment:
a `get` method that doesn't return anything? I would change that to `set` if what it does is set the internal field